app/admin/api_users.rb

```diff
@@ -0,0 +1,45 @@
+ActiveAdmin.register ApiUser do
+  permit_params :name
+
+  index do
+    selectable_column
+    id_column
+    column :name
+    column :created_at
+    actions
+  end
+
+  show do |f|
+    attributes_table do
+      row :name
+      row :created_at
+    end
+  end
+
+  form do |f|
+    f.inputs do
+      f.input :name
+    end
+    f.actions
+  end
+
+  controller do
+    def create
+      name = params[:api_user][:name]
+
+      token = SecureRandom.hex(40)
+      token_digest = Digest::SHA256.hexdigest(token)
```

Comment:
I'm not sure if this is secure enough. Instead of storing the API token in the DB, I'm storing its sha256 hash. But bcrypt, for example stores a salted hash, after hashing many times. We can't do that for API keys because salt causes the hash to be different even for the same password (or API key). So to authenticate a user by using bcrypt, we'd need to do a full table scan, checking each token digest individually. bcrypt has been deliberately designed so that checking takes a significant amount of CPU time.